mti_nma/steps/nma/nma_viz.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import seaborn as sb
+import matplotlib.pyplot as plt
+import subprocess
+
+
+def draw_whist(w):
+    """
+    Draws a histogram figure of the eigenvalues
+
+    Parameters
+    ----------
+    w: Numpy 1D array
+        An array of eigenvalue values
+    Returns
+    -------
+    fig: Matplotlib Figure object
+        Figure containg a histogram of eigenvalues
+    """
+
+    plt.clf()
+    fig = plt.figure()
+
+    # set binning
+    minval = min(w) - 0.5
+    maxval = max(w) + 0.5
+    if len(w) < 20:
+        N = int(max(w) + 2)
+    else:
+        N = 30
+    bins = np.linspace(minval, maxval, N)
+
+    sb.distplot(w, kde=False, bins=bins)
+    plt.xlabel('Eigenvalues (w2*m/k)')
+    plt.ylabel('Counts')
+
+    return fig
+
+
+def color_vertices_by_magnitude(path_input_mesh, path_vmags, mode, path_output):
+    """
+    Creates blender file with mesh vertices colored by eigenvector 
+    magnitudes for a given mode and saves as .blend file
+    The Python script to do this must be run in Blender, so we open Blender
+    in bash and run the python script there.
+
+    If your local copy of Blender is in a different location than the current
+    path listed, change the filepath at the start of the `bl` string to 
+    your own Blender path.
+
+    The `-b` flag runs Blender headlessly (doesn't open the app GUI) and the
+    `-P` flag tells Blender you want to run the python script whose filepath is
+    provided after this flag. The `--` indicated to blender that the arguments
+    following it are arguments for the python script, not for Blender.
+
+    Parameters
+    ----------
+    path_input_mesh: str
+        Filepath to input mesh to be colored (.stl)
+    path_vmags: str
+        Filepath to input magnitudes of eigenvector used to color mesh (.npy)
+    mode: int
+        Index of mode whose eigenvector magnitudes will be used to color mesh
+    path_output: str
+        Filepath to output file of colored mesh object (.blend)
+    """
+
+    bl = "/Applications/Blender.app/Contents/MacOS/Blender -b -P "
+    psc = "/Users/juliec/mti/mti_nma/mti_nma/steps/nma/color_vertices.py -- "
```

Comment:
Now that I am fully reading through this PR, I see how you are using this. It may be worth it (probably is) to move the `color_vertices.py` file to the bin folder and add it to you `console_scripts` setup.py, that way you could change this to simply be `nma_color_vertices` or similar to how you would call any command line function and then regardless of where this is running, distributed or local machine, it wouldn't matter it would simply look for that bin script install.

Reply:
I'm not sure I really follow this...

Reply:
Would you mind explaining this alternative structuring in more detail?

Reply:
For documenting purposes: from today in person, we moved that python file to the `bin` directory of the project and added a reference to it to the `console_scripts` section in `setup.py`. By doing this what we accomplished was that after a user installs the package, the system will know that when the user calls `color_vertices` in their terminal it will find the script, wherever it lives.

So instead of a hard coded path in the code to a location that works for you this entire thing can (I think we already did) be switched to simply `"color_vertices --"` which will work for everyone running this step 💯